node_helper.js

```diff
@@ -589,12 +589,14 @@ module.exports = NodeHelper.create({
     For NASA API usage without an API key, there are hourly limits of about 1,000 requests. 
     */
   processNasaData: function (config, data) {
+    // filter for images, since the API also returns videos
+    const filteredImages = data?.collection?.items.filter((item) => item?.data?.[0]['media_type'] === 'image');
     let images = [];
-    for (const image of data.collection.items) {
-      if (image.links && image.links.length > 0) {
+    for (const image of filteredImages) {
+      if (image?.links && image?.links?.length > 0) {
```

Comment:
Some users have older versions of node that don't support the `?.` syntax, please remove them

Reply:
Hi @kolbyjack! Just fixed that. Have a nice day!